src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -235,6 +226,35 @@ public Response getPoliciesForCustomer() {
         }
       });
 
+      // Last triggerTime
+      String ids = page.stream().map(p -> p.id.toString()).collect(Collectors.joining(","));
+      try {
+        List<Alert> list = engine.findLastTriggered(ids, true, user.getAccount());
+        // The list contains multiple alerts per trigger, so find the newest
+        Map<String, Long> times = new HashMap<>(page.size());
+        for (Alert a : list) {
+          String tid = a.trigger.id;
+          if (!times.containsKey(tid)) {
+            times.put(tid, a.ctime);
+          } else {
+            long stored = times.get(tid);
+            if (stored < a.ctime) {
+              times.put(tid, a.ctime);
+            }
+          }
+        }
+        // Now loop over the map and transfer the times to the policies
+        page.forEach(p -> {
+          if (times.containsKey(p.id.toString())) {
+            long lastTriggerTime = times.get(p.id.toString());
+            p.setLastTriggered(lastTriggerTime);
```

Comment:
unix timestamp?

Reply:
Yes.
I can also convert that to Timestamp. But as we had the discussion that Unix ts may be better, I did it this way.

Reply:
Unix timestamp is good, thanks!